frontend/device/kindle/device.lua

```diff
@@ -1188,7 +1224,8 @@ local pw4_set = Set { "0PP", "0T1", "0T2", "0T3", "0T4", "0T5", "0T6",
                   "16Q", "16R", "16S", "16T", "16U", "16V" }
 local kt4_set = Set { "10L", "0WF", "0WG", "0WH", "0WJ", "0VB" }
 local koa3_set = Set { "11L", "0WQ", "0WP", "0WN", "0WM", "0WL" }
-local pw5_set = Set { "1LG", "1Q0", "1PX", "1VD", "219", "21A", "2BH", "2BJ" }
+local pw5_set = Set { "1LG", "1Q0", "1PX", "1VD", "219", "21A", "2BH", "2BJ", "2DK" }
+local scribe_set = Set { "22D", "25T", "23A", "2AQ", "2AP", "1XH", "22C" }
```

Comment:
Just for my curiosity, how did you get all of them?

Reply:
Depends on the FW version, currently it's in `/etc/deviceTypes.conf` (a similar file has existed throughout the FW history, but it wasn't necessarily always this one).

There were two new PW5 ids in 5.14.2, one in 5.14.3, and the Scribe appeared in 5.15.1 (i.e., it'll probably ship on 5.15.0).